docs/cli.md

```diff
@@ -1,17 +1,17 @@
 # NEAR AI CLI
 
-The NEAR AI CLI allows you to interact with NEAR AI's services to [create agents](./agents/quickstart.md) or [train and test models](./models/home.md). Let's quickly review how to install it and login with your NEAR account.
-
-!!! tip
-    Do you want to build an agent? Install the CLI, login, and then check our [**Agent Quickstart Guide**](./agents/quickstart.md).
-
-!!! info
-    Remember that you can use NEAR AI from the [Web Hub](https://hub.near.ai), the NEAR AI CLI is targeted at developers who want to build their own agents or train models on NEAR AI.
+NEAR AI CLI allows you to [create and deploy agents](./agents/quickstart.md), [train and test models](./models/home.md), and more!
 
 ---
 
 ## Installing NEAR AI CLI
 
+!!! warning
+    Requires Python version **`3.11`** _(Currently does not work with `3.12` or `3.13`)_
+
+!!! tip
+     Although not required, this project uses [uv](https://docs.astral.sh/uv/) for Python package management and it is suggested that you use it as well if you would like to make contributions to this project.
```

Comment:
is this really necessary?

Reply:
Yep, I think 3.9 & 3.10 work as we have CI that tests these versions but I had someone at a hackathon who had 3.9 and was unable to use it until they upgraded to 3.11. This may have been a windows related issue though.... I can update this to be 3.9-3.11 and see if we get any reports on it.